playbooks/gitea.yaml

```diff
@@ -13,6 +13,9 @@
     gitops_password: "12341234"
     gitops_repo_name: "test-repo"
 
+    external_protocol: "http"
```

Comment:
If making a change, worth adding these with these default values to role's defaults?

Reply:
Not sure exactly what this means sorry!

Reply:
Seems resolved by the line below.